_posts/2018-07-20-Rust-1.27.2.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Announcing Rust 1.27.2"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.27.2. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.27.2 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.27.2][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1272-2018-07-20
+
+## What's in 1.27.2 stable
+
+This patch release fixes a bug in the borrow checker verification of `match` expressions. This bug
+was introduced in 1.26.0 with the stabilization of [match ergonomics]. Specifically, it permitted
+code which took two mutable borrows of the `bar` path at the same time.
```

Comment:
Based on this sentence I would expect to see something called `bar` and some mutable borrows in the code below.

Reply:
Heh, thanks. Maybe copy/pasting the file from 1.26.2 wasn't the best idea...